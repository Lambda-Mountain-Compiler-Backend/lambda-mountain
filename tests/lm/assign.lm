
main := print-s(tail(
   (local a)
   (local b)
   (local c)
   (set a 123)
   (set b 'abc)
   (set c 'xyz)
   (Assignment a b c)
));
