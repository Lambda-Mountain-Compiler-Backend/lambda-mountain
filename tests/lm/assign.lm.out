(((Assignment 123) abc) xyz)
